external-provider-ui/webpack.config.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
+const webpack = require('webpack');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ModuleFederationPlugin = require("webpack/lib/container/ModuleFederationPlugin");
+const getModuleFederationConfig = require('@jahia/webpack-config/getModuleFederationConfig');
+const packageJson = require('./package.json');
+const {CycloneDxWebpackPlugin} = require('@cyclonedx/webpack-plugin');
+
+/** @type {import('@cyclonedx/webpack-plugin').CycloneDxWebpackPluginOptions} */
+const cycloneDxWebpackPluginOptions = {
+    specVersion: '1.4',
+    rootComponentType: 'library',
+    outputLocation: './bom'
+};
+
+module.exports = (env, argv) => {
+    let config = {
+        entry: {
+            main: [path.resolve(__dirname, 'src/javascript/index.js')]
+        },
+        output: {
+            path: path.resolve(__dirname, 'src/main/resources/javascript/apps/'),
+            filename: 'edp.bundle.js',
+            chunkFilename: '[name].edp.[chunkhash:6].js'
```

Comment:
do we need a diff bundle name for external-provider-ui?

Reply:
yes, thanks, it's better with a different name